### PR TITLE
docs(nuget): add hint for protocol version in NuGet.config

### PR DESCRIPTION
**Pull request by fgreinacher**
## Changes

Update NuGet docs to mention how to define the protocol version for feeds in `NuGet.config` files. 

## Context

I forgot to update this documentation in https://github.com/renovatebot/renovate/pull/5757. Funnily this was nearly four years ago and nobody noticed :) 

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

:hammer_and_pick: with :heart: by [Siemens](https://opensource.siemens.com/)

### CODE REVIEW

**Review by viceice — DISMISSED** [review]

File: docs/usage/nuget.md
Comment (HonkingGoose) [inline docs/usage/nuget.md:81]:
```suggestion
If a `v3` feed URL does not end with `index.json`, you must specify the version explicitly.
```

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

File: docs/usage/nuget.md
Comment (fgreinacher) [reply docs/usage/nuget.md]: Thanks 😊

**Review by viceice — APPROVED** [review]

**Review by HonkingGoose — COMMENTED** [review]
I'm happy with the docs and layout. I'll let the maintainers review for technical accuracy.

**Pull request merged**
